src/hooks/useLatestGitHubRelease.ts

```diff
@@ -38,7 +39,7 @@ const useLatestGitHubRelease = () => {
     const data = await response.json()
     const latestVersion = data.tag_name.replace('v', '')
 
-    if (semverRegex.test(latestVersion) && currentVersion !== latestVersion) {
+    if (semverRegex.test(latestVersion) && currentVersion && semverCompare(latestVersion, currentVersion) > 0) {
```

Comment:
Nit: This check `semverRegex.test(latestVersion) ` should probably happen just before and if it fails, give a console.warn but it's not vital.

Reply:
The regex is there to ensure that we don't check for release candidates. I don't see the reason to have a warning for that.

Reply:
The warning would be "Does not follow standard semver syntax" in the cases we're run running a non-standard version (like you said) which makes it clear when developing (again it's just a Nit, I'd find this useful but maybe you not)

Reply:
hm, why would you care whether there's a non-standard version available on GitHub while you're developing on your local wallet?

Reply:
It's more independent of that: there may be a scenario when someone is running a dev version and has forgotten (maybe they self-built and it's been months) - otherwise I agree when developing you know X) I'm coming from a place of "If the semver string is not standard, show something" where I'm not assuming why, but the fact it can happen... Reporting it's not standard sounds useful to me because it becomes clear immediately this is not a standard release, *for whatever reason*

Reply:
> Reporting that it's not standard sounds useful to me because it becomes clear immediately this is not a standard release

I don't really see any value as to why there should be a warning in the console when the latest release on GitHub is not one that the user should be concerned about. I'll bypass this nit, if that's ok, and merge :)